Figure out CI problems

Builds for me locally on OSX, but AppVeyor and Travis report an error:

```
/home/travis/.nuget/packages/fsharp.net.sdk/1.0.0-beta-060000/build/FSharp.NET.Core.Sdk.targets(137,9): error MSB3073: The command ""/home/travis/build/fable-compiler/fable-elmish/dotnetsdk/sdk/1.0.1/../../dotnet" compile-fsc "@/home/travis/build/fable-compiler/fable-elmish/src/elmish/node_modules/fable-powerpack/obj/Debug/netstandard1.6/dotnet-compile.rsp" " exited with code 1. [/home/travis/build/fable-compiler/fable-elmish/src/elmish/node_modules/fable-powerpack/Fable.PowerPack.fsproj]
/home/travis/build/fable-compiler/fable-elmish/dotnetsdk/sdk/1.0.1/Sdks/Microsoft.NET.Sdk/build/Microsoft.NET.Sdk.targets(92,5): error : Cannot find project info for '/home/travis/build/fable-compiler/fable-elmish/src/elmish/node_modules/fable-powerpack/Fable.PowerPack.fsproj'. This can indicate a missing project reference. [/home/travis/build/fable-compiler/fable-elmish/src/elmish/Fable.Elmish.fsproj]
```

* fable-elmish version: (master)
* fable-compiler version: (narumi)
* fable-core version: (narumi)
* Operating system: linux, windows

--- a/src/elmish-react/package.json
+++ b/src/elmish-react/package.json
@@ -15,6 +15,6 @@
   "author": "Eugene Tolmachev",
   "license": "Apache-2.0",
   "devDependencies": {
-    "fable-react": "^1.0.0-narumi-1"
+    "fable-react": "^1.0.0-narumi-4"
   }
 }

--- a/src/elmish/package.json
+++ b/src/elmish/package.json
@@ -16,6 +16,6 @@
   "author": "Eugene Tolmachev",
   "license": "Apache-2.0",
   "devDependencies": {
-    "fable-powerpack": "^1.0.0-narumi-1"
+    "fable-powerpack": "^1.0.0-narumi-2"
   }
 }

